libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -266,7 +269,7 @@ class AC_WPNav
 
     // references and pointers to external libraries
     const AP_InertialNav&   _inav;
-    const AP_AHRS_View&     _ahrs;
+    AP_AHRS_View&     _ahrs;
```

Comment:
isn't this a singleton now?

Reply:
@magicrub  It might be a singleton in master.  I PR'd against 3.6 because I initially wanted to get this running in 3.6 for our user base.  I knew if I got this integrated properly that it would be very popular among the heli community.